test/project/project.godot

```diff
@@ -12,7 +12,7 @@ config_version=5
 
 config/name="GDExtension Test Project"
 run/main_scene="res://main.tscn"
-config/features=PackedStringArray("4.3")
+config/features=PackedStringArray("4.4")
```

Comment:
This unrelated change should be removed. `master` does depend on 4.4 at this point, and this could be updated, but this isn't the PR to do it (especially if we decide to cherry-pick this change to an earlier version)

Reply:
Totally, just committed that by accident!